### PR TITLE
feat : auth-page

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- please review 
-[Login page](https://eh2vm7.csb.app/auth/login.html) 
- [logout page](https://eh2vm7.csb.app/auth/logout.html)
- [signup page](https://eh2vm7.csb.app/auth/signup.html)

### CODE REVIEW

File: auth/login.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+.inp-width{
+    width: 20%;
+    margin: 0.5rem;
+    height: 30px;
+}
+.login-head{
+    display: flex;
+}
+.x{
```
Comment (MaruthiPothuganti) [inline auth/login.css:61]: Its good to have a understandable class names

File: auth/login.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    padding-left: 10rem;
+}
+.login{
+    width: 50%;
+}
+.btn{
+    background-color: blue;
+    width: 10%;
+    height: 50px;
```
Comment (MaruthiPothuganti) [inline auth/login.css:70]: Good to use Rems instead of Pixels. as px hinders responsiveness without media queries

File: auth/signup.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <a href="./wishlist.html" class="a-link">
+        <i class="fa-solid fa-heart"></i>
+            </a>
+        </div>
+    </nav>
+    <div class="signup">
+        <div class="signup-bar">
+            <label for="Firstname">FirstName</label>
+            <input class="input-btn" type="text" name="Firstname">
+            <label for="Lasttname">LastTname</label>
```
Comment (akaysing) [inline auth/signup.html:33]: please correct the spelling of LastName

File: auth/login.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}
+.nav{
+    display: grid;
+    background-color: #9bb7d4;
+    padding: 0.8rem;
+}
+.name-logo{
+    grid-column: 1/2;
+    grid-row: 1/2;
+    font-size: 25px;
```
Comment (akaysing) [inline auth/login.css:13]: i would suggest using rem everywhere instead of px

File: auth/login.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    </nav>
+    <div class="login-bar">
+        <div class="login-head">
+        <h3>Login</h3>
+        <a href="/index.html" class="x">X</a>
+       </div>
+        <label for="name">Input</label>
+        <input type="text" name="name" class="inp-width">
+        <label for="password">Password</label>
+        <input type="text" name="password" class="inp-width">
```
Comment (MaruthiPothuganti) [inline auth/login.html:37]: better to have type as password for password input field

File: auth/logout.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    text-decoration: none; 
+    color: black;
+}
+.flex-main{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 15rem;
+}
+h2{
```
Comment (MaruthiPothuganti) [inline auth/logout.css:43]: avoid styling using tag names. good to have a classname instead

File: auth/logout.css [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    background-color: #9bb7d4;
+    border-radius: 1rem;
+    width: 20%;
+    text-align: center;
+    height: 35px;
```
Comment (MaruthiPothuganti) [inline auth/logout.css:48]: Same as above, Can use REM

File: auth/logout.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (MaruthiPothuganti) [inline auth/logout.html:7]: Better to have a title. Here as **Logout**

**Review by akaysing — COMMENTED** [review]
u can have better bg color for login and create account buttons will make them look good

File: auth/signup.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+     background-color: blue;
+     width: 10%;
+     height: 50px;
+     border-radius: 5px;
+     margin-top: 1rem;
+}
+.signup-bar{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (MaruthiPothuganti) [inline auth/signup.css:52]: These styles can be made common and class name can be used, instead of hard coding everywhere.

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested some changes. Please look into them.  And 
a Suggestion: Instead of CSB links, you can deploy it using netlify. It can generate deploy previews automatically after every PR.

**Pull request merged**